resources/views/events/admin/edit.blade.php

```diff
@@ -106,6 +106,31 @@
             </div>
         </label>
 
+        <label class="mt-6 block md:flex">
+            <div class="md:w-1/3">
+                <div class="font-semibold text-black mb-2">Last cancel date</div>
+                <div class="text-gray-700 text-sm">The last date attendees are able to cancel their spot. Not required.</div>
+            </div>
+
+            <div class="md:w-2/3">
+                <div>
+                    <input type="datetime-local" name="last_cancel_date" class="form-input w-full"
+                           required value="{{ old('last_cancel_date', \App\Utils\Date::fromString($event->last_cancel_date)->tz(config('app.timezone'))->format('Y-m-d\\TH:i')) }}">
```

Comment:
This causes `Call to a member function tz() on null` when editing an event where last cancel date is not set

Reply:
Fixed in 86471e2